openfl-workspace/torch_cnn_histology/src/dataloader.py

```diff
@@ -37,9 +37,14 @@ def __init__(self, data_path, batch_size, **kwargs):
         """
         super().__init__(batch_size, random_seed=0, **kwargs)
 
-        _, num_classes, X_train, y_train, X_valid, y_valid = load_histology_shard(
-            shard_num=int(data_path), **kwargs)
+        try:
+            int(data_path)
+        except:
+            raise ValueError("Expected `%s` to be representable as `int`.", data_path)
```

Comment:
In the error message, specify that this is for the shard number, used to define the data path for the collaborator.

Reply:
Addressed in ca90679af5ca4788e1bc8886dda550dd34f57a0d.